myuw/templates/base.html

```diff
@@ -265,6 +261,7 @@ <h2 id="categories" class="sr-only">Categorized Resources</h2>
                         <li class="footer-links-list-item"><a href="//www.washington.edu/online/terms/">Terms</a></li>
                         <li class="footer-links-list-item"><a href="//www.washington.edu/online/privacy/">Privacy</a></li>
                         <li class="footer-links-list-item visible-xs-inline"><a href="/logout">Sign Out</a></li>
+                        <li class="footer-links-list-item visible-sm-inline"><a href="#" class="myuw-muted" data-toggle="modal" data-target="#optout_modal">Switch back to the old MyUW</a></li>
```

Comment:
I'm not seeing this render in the UI. I can see the code in the inspector (emulating a mobile device), but nothing visible in the UI. Is this an issue with my set up, or do you see the same?

Reply:
I'm not sure why I put one as visible-xs-inline and the other link as visible-sm-inline. I just changed them to both be visible-sm-inline, so both links should show up below about 1000px wide.
![myuw__home](https://user-images.githubusercontent.com/6953502/32743071-3e8000d4-c860-11e7-9a6c-b503906ea764.jpg)